backend/tests/services/execution/test_dispatcher_sse_request_id.py

```
@@ -0,0 +1,181 @@
+    async def __anext__(self):
+        if not self._events:
+            raise StopAsyncIteration
+        return self._events.pop(0)
+
+
+def _build_fake_openai_module(calls: dict):
+    class _FakeResponses:
+        async def create(
+            self,
+            model,
+            input,
```

Ruff (0.15.7)

[error] Function argument `input` is shadowing a Python builtin

(A002)